_episodes/02-shell-basics.md

```diff
@@ -1,31 +1,418 @@
+- "How do I create, delete, move and rename files and folders?"
+- "How do I create and edit text files?"
+keypoints:
+- "Use `cd`, `pwd` and `ls` to navigate the file system and inspect its contents"
+- "Use `cp`, `mv` and `rm` to copy, move and remove files or directories"
+- "Use `mkdir` and `touch` to create new empty directories and files respectively"
+- "The location of a file or folder, i.e., the *path* to the file or folder can be relative or absolute. An absolute path always starts from the root directory (`/`)"
+- "Text files can be created and edited from the command-line using an editor like `nano` or `vim`"
 ---
 
-Topics: 
+The command-line or **shell** is another way of interacting with a computer,
+just like the graphical interface (windows, pointer, icons, toolbars and menus)
+that you may be more familiar with.
+The difference is that rather than
+clicking on buttons, menu items and check-boxes, or entering text into graphical boxes,
+you provide instructions to the computer by
+issuing commands to it.
 
-Recommendation: for a novice audience, go through 1,2,3 in 
-the [shell-novice lesson](http://swcarpentry.github.io/shell-novice/)
+As a few examples of tasks you might do on your computer,
+consider:
 
-Also convey: 
-The shell is more than a way to look at files or folders. It's also a way to run programs.
+* Creating new files or folders
+* Navigating between folders on your computer and viewing their contents
+* Moving, renaming, deleting or copying files and folders
 
-Programs may expect data as input and may produce data as output. The shell lets us 
-control which programs to run on which inputs and what to do with the output.
+You are probably familiar with using a graphical file browser
+such as **Windows Explorer** (Windows) or **Finder** (Mac OS X)
+for doing the above tasks:
+The same tasks can be performed on the command-line,
+using a few simple commands:
 
-Maybe use a shell program like sleep or grep as an example, or another simple executable.  
+| Command 	  			| Action
+------------------------|-------------------------------------------
+| `pwd`		  			| Print working directory
+| `cd DIR` 				| Change directory to `DIR`	
+| `ls`					| List contents of current directory
+| `mkdir DIR`			| Make new empty directory `DIR`
+| `touch FILE`          | Make new empty file `FILE`
+| `cp SRC DEST`			| Copy file `SRC` into file `DEST`
+| `cp -r SRC DEST`		| Copy directory `SRC` into directory `DEST`
+| `rm FILE`				| Remove (delete) file `DIR`
+| `rm -r DIR`			| Remove (delete) directory `DIR`
+| `mv SRC DEST`         | Move the file (or directory) `SRC` into the directory `DEST`.
 
-Then do part 6 of the shell lesson.  
+In this part of the lesson we'll explore
+and familiarize ourselves with such commands.
 
-For a more advanced audience, just do part 6, so that you can use 
-the script later in the workshop.  
\ No newline at end of file
+When you login to the cluster,
```

Comment:
add a reference to the next section of the material as right here we don't `ssh` onto the machine.